CONTRIBUTING.md

```diff
@@ -1,95 +1,111 @@
 
-* All contributions must NOT add any SwiftLint warnings or errors. There is a GitHub action setup for any PRs to `dev`, and Xcode will show any warnings/errors.
+- All contributions must NOT add any SwiftLint warnings or errors. There is a GitHub action setup for any PRs to `dev`, and Xcode will show any warnings/errors.
 
 # About the Project
+
 This app was created for [Hacktoberfest](https://hacktoberfest.com/), to help beginners in iOS dev contribute to open source. It is an app to help keep track of your car maintenance activites. This project uses [Firebase](https://firebase.google.com), [Firestore](https://firebase.google.com/products/firestore), and Sign in With Apple.
 
 ### Project Status
+
 This app will be deployed on the Apple App Store, available for iOS 17.0 or later, [Mikaela Caron](https://github.com/mikaelacaron) (the maintainer) will upload it to the App Store after Hacktoberfest.
 
 # Getting Started
+
 ## Prerequisites
-* Download Xcode 16.0 or later
-* Install [SwiftLint](https://github.com/realm/SwiftLint) onto your machine via [Homebrew](https://brew.sh/)
-   * This is not a requirement, but is preferred.
+
+- Download Xcode 16.0 or later
+- Install [SwiftLint](https://github.com/realm/SwiftLint) onto your machine via [Homebrew](https://brew.sh/)
+  - This is not a requirement, but is preferred.
+
 ```sh
 brew install swiftlint
 ```
-* Set your Xcode settings correctly:
-   * Open Xcode Settings `Cmd + ,`
-   * Text Editing
-   * Indentation tab
-   * Prefer Indent using Spaces
-   * Tab Width: 4
-   * Indent Width: 4
-   * Xcode 16+
-      * Check "Prefer Settings from EditorConfig"
+
+- Set your Xcode settings correctly:
+  - Open Xcode Settings `Cmd + ,`
+  - Text Editing
+  - Indentation tab
+  - Prefer Indent using Spaces
+  - Tab Width: 4
+  - Indent Width: 4
+  - Xcode 16+
+    - Check "Prefer Settings from EditorConfig"
 
 ## Start Here
-* **Fork** this repo
-* **Clone** the repo to your machine (do **not** open Xcode yet)
-* In the same folder that contains the `Basic-Car-Maintenance.xcconfig.template`, run this command, in Terminal, to create a new Xcode configuration file (which properly sets up the signing information)
+
+- **Fork** this repo
+- **Clone** the repo to your machine (do **not** open Xcode yet)
+- In the same folder that contains the `Basic-Car-Maintenance.xcconfig.template`, run this command, in Terminal, to create a new Xcode configuration file (which properly sets up the signing information)
 
 ```sh
 cp Basic-Car-Maintenance.xcconfig.template Basic-Car-Maintenance.xcconfig
 ```
 
-* Open Xcode
+- Open Xcode
+
+- In the `Basic-Car-Maintenance.xcconfig` file, fill in your `DEVELOPMENT_TEAM` and `PRODUCT_BUNDLE_IDENTIFIER`.
+  - You can find this by logging into the Apple Developer Portal
+  - This works with both free or paid Apple Developer accounts. Do **NOT** run this app on a real device due to issues with the Sign in With Apple capability.
```

Comment:
Please go through the rest of this document, and change back all of the `-` back to `*`